static/bidder-params/flatads.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Flatads Adapter Params",
+  "description": "A schema which validates params accepted by the Flatads adapter",
+  "type": "object",
+  "properties": {
+    "token": {
+      "type": "string",
+      "description": "Token of the publisher"
+    },
+    "publisherId": {
+      "type": "string",
+      "description": "Flatads Publisher Id"
+    }

```

Comment:
Optional: you can add `"minLength": 1` to make sure values are not empty string, if empty values are not expected.